docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1168,6 +1173,11 @@ paths:
           description: show subscription status
           type: boolean
           default: false
+        - name: readers
+          in: query
+          description: |
+            The username or service name that has explicit read authorization rights.
+          type: string
```

Comment:
```suggestion
          collectionFormat: multi
          type: array
          items:
            type: string
```

Reply:
Is array more appropriate to describe a list of readers?

Reply:
I've copied it over from the definition of `event_type` param: https://github.com/zalando/nakadi/blob/master/docs/_data/nakadi-event-bus-api.yaml#L1127-L1137